src/app/app-routing.module.ts

```diff
@@ -159,13 +168,51 @@ export const routes: Routes = [
                 resolve: { [RoutingResolvers.METADATA_BLOCK_KEY]: blockMetadataResolver },
             },
             {
-                path: "",
+                path: `${ProjectLinks.URL_FLOW_DETAILS}/:${ProjectLinks.URL_PARAM_FLOW_ID}`,
+                component: DatasetFlowDetailsComponent,
                 canActivate: [AuthenticatedGuard],
+                runGuardsAndResolvers: "always",
+                resolve: {
+                    [RoutingResolvers.FLOW_DETAILS_ACTIVE_TAB_KEY]: activeTabResolver,
+                    [RoutingResolvers.FLOW_DETAILS_KEY]: flowDetailsResolver,
+                    [RoutingResolvers.DATASET_INFO_KEY]: datasetInfoResolver,
+                },
                 children: [
                     {
-                        path: `${ProjectLinks.URL_FLOW_DETAILS}/:${ProjectLinks.URL_PARAM_FLOW_ID}/:${ProjectLinks.URL_PARAM_CATEGORY}`,
-                        component: DatasetFlowDetailsComponent,
+                        path: FlowDetailsTabs.HISTORY,
+                        component: FlowDetailsHistoryTabComponent,
+                        runGuardsAndResolvers: "always",
+                        resolve: {
+                            [RoutingResolvers.FLOW_DETAILS_HISTORY_KEY]: flowDetailsSummaryResolver,
```

Comment:
What do you think about moving the `flowDetailsSummaryResolver` to the parent path?
It seems it can be loaded only once (not counting the timer case).

Reply:
I don't agree - well, you need to check the state in accordance with the timer - it can change. For example "RUNNING"=>"FINISHED"